.github/workflows/karisma-stories.floeproject.org__production.yml

```diff
@@ -0,0 +1,39 @@
+name: karisma-stories.floeproject.org-staging__production
+
+on:
+  push:
+    branches:
+      - stories-karisma-production
```

Comment:
This branch won't exist until #64 has been dealt with

Reply:
I'll have to wait on that one then because I can't deploy from this repo contents from that repo. It's better if everything is in the same repo.